src/Components/Calculator.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable react/jsx-no-bind */
+/* eslint-disable  react/prefer-stateless-function */
+import React, { useEffect, useState } from 'react';
+import Row from './RowCalcul';
+
+export default function Calculator() {
```

Comment:
- [ ] I would like to recommend you use the `arrow function` instead of the `normal function `in your code since we use ES6  :+1: